src/Modules/CalcDefence.lua

```diff
@@ -2393,13 +2396,18 @@ function calcs.defence(env, actor)
 					})
 				end
 			end
+
+			local takenMult = output[damageType.."TotalHitPool"] / output[damageType.."MaximumHitTaken"] / enemyDamageMult
+
 			t_insert(breakdown[damageType.."MaximumHitTaken"], s_format("Total Pool: %d", output[damageType.."TotalHitPool"]))
```

Comment:
Personally this
```
Total Pool: 1,000
/ 0.50 (modifiers to damage taken)
/ 0.50 (modifiers to enemy damage)
= 4,000
```
reads as if the total pool is 4000, not the max hit. Should be possible to make this clearer with different wording.

Reply:
I just used same wording as EHP is already using I think its fine

Reply:
Maybe add (maximum hit) to the last line? Or something similar.

Reply:
Well the column already implies that the result is max hit I think so thats probs unnecessary. Maybe I can just swap Total Pool: 1000 to 1000 (Total Pool)

Reply:
![image](https://user-images.githubusercontent.com/5115805/209471882-bd6bda14-857b-42ca-b7f2-28b7fc58efc3.png)

Like this is how it looks like when you hover. I think the max hit being result of the calculation is quite clear at least, idk